config/application.rb

```diff
@@ -33,5 +33,11 @@ class Application < Rails::Application
 
       g.fallbacks[:rspec] = :test_unit
     end
+
+    # Configure webpack
+    config.webpack.config_file = "config/webpack.js"
+    config.webpack.output_dir  = "public/assets/webpack"
+    config.webpack.public_path = "assets/webpack"
+    config.webpack.dev_server.enabled = false
```

Comment:
With this PR I'm not using the webpack-dev-server, but I simply call `webpack --watch --config config/webpack.js`. I know that this is not ideal but adding the dev server seemed like something that could be done on another PR instead of this one.